### DISCUSSION TITLE
Question about installation

### DISCUSSION THREAD

**Discussion by erickaufmann16** [JSQLParser/JSqlParser#2072] [General]
Hey, I am new to JSQLParser and having difficulty start using it.

From the doc here: https://jsqlparser.github.io/JSqlParser/usage.html, 
it says:

git clone https://github.com/JSQLParser/JSqlParser.git
cd jsqlformatter
mvn install

But I can't find the jsqlformatter directory. Wish there is doc with more details.

Eric

**Comment by manticore-projects** [comment]
Greetings!

Thank you for the notification, it was an (copy'n paste) related error which I have fixed.
Corrected instructions are:

```bash
    git clone --depth 1 https://github.com/JSQLParser/JSqlParser.git
    cd JSqlParser/
    mvn install
```

Should also be shown on the website within a few minutes.

**Reply by erickaufmann16** [reply]
Thanks.
